packages/bananagram-common/src/example.test.ts

```diff
@@ -0,0 +1,6 @@
+import { createPieces, numberOfBanangramTokens } from './example';
+
+it('should do something', () => {
```

Comment:
Maybe this test name could be a bit more descriptive?

Reply:
removed example code b1d854c